src/app/file-explorer.js

```diff
@@ -55,8 +66,11 @@ function fileExplorer (appAPI, files) {
       }
     },
     formatSelf: function (key, data) {
-      return yo`<label class=${data.children ? css.folder : css.file}
+      var isRoot = data.path.indexOf('/') === -1
+      return yo`<label class="${data.children ? css.folder : css.file}"
         data-path="${data.path}"
+        style="${isRoot ? 'font-weight:bold;' : ''}"
+        isfolder=${data.children !== undefined}
```

Comment:
it is used by the `files.rename` https://github.com/ethereum/browser-solidity/pull/581/files#diff-6b0a773f8389fe76bf78a69d77e8d9b3R202